src/components/calcite-value-list/calcite-value-list.tsx

```diff
@@ -202,6 +203,38 @@ export class CalciteValueList {
 
   getItemData = getItemData.bind(this);
 
+  keyDownHandler = (event) => {
+    const handleElement = event.composedPath().find((item) => {
+      return item.dataset && item.dataset.jsHandle;
```

Comment:
nitpick: I think we can use optional chaining now with latest TS.

Reply:
I believe that comes with version 3.7, and seems we're using 3.6.3. Is there an update in master that bumps this?

Reply:
We're on 3.7.2 on master.

Reply:
sweet, merged it in. optional chaining ftw.